biggerquery/cli.py

```diff
@@ -130,35 +134,64 @@ def read_project_name_from_setup() -> Optional[str]:
 
 def build_project_name_description(project_name: str) -> str:
     if project_name is None:
-        return 'Project name not found in the project_setup.PROJECT_NAME.'
+        return ''
     else:
         return 'Project name is taken from project_setup.PROJECT_NAME: {0}.'.format(project_name)
 
 
-def find_root_package(project_name: Optional[str], root: Optional[str]) -> Path:
+def find_root_package(project_name: Optional[str], project_dir: Optional[str]) -> Path:
     """
     Finds project package path. Tries first to find locatin in project_setup.PROJECT_NAME,
     and if not found then by making a path to the `root` module
 
-    @param root: Path to the root package of this project, used only when PROJECT_NAME not set
+    @param project_dir: Path to the root package of a project, used only when PROJECT_NAME not set
     @return: Path
     """
     if project_name is not None:
         return Path(project_name)
     else:
-        print('The project_setup.PROJECT_NAME not found. Looking for the --root.')
-        root_module = import_module(root)
+        print(f'The project_setup.PROJECT_NAME not found. Looking for the root module in {project_dir}')
+        root_module = import_module(project_dir)
         return Path(root_module.__file__.replace('__init__.py', ''))
 
 
-def cli_run(root_package: Path,
+def _decode_version_number_from_file_name(file_path: Path):
+    if file_path.suffix != '.tar':
+        raise ValueError(f'*.tar file expected in {file_path.as_posix()}, got {file_path.suffix}')
+    if not file_path.is_file():
+        raise ValueError(f'File not found: {file_path.as_posix()}')
+
+    split = file_path.stem.split('-', maxsplit=1)
+    if not len(split) == 2:
+        raise ValueError(f'Invalid file name pattern: {file_path.as_posix()}, expected: *-{{version}}.tar, for example: image-0.1.0.tar')
+    return split[1]
+
+
+def import_deployment_config(deployment_config_path: str):
+    spec = importlib.util.spec_from_file_location("deployment_config", deployment_config_path)
```

Comment:
Use single quotes or double quotes everywhere possible. In our case single quote.